apps/authorizer/lib/policies/behaviour.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Authorizer.Policies.Behaviour do
+  @moduledoc """
+  A policy is a set of verifications to make sure that a subject can do such action.
+  """
+
+  @doc "Return the policy description"
+  @callback info() :: String.t()
+
+  @doc "Runs the input validations"
+  @callback validate(conn :: Plug.Conn.t()) :: {:ok, context :: map()} | {:error, atom()}
```

Comment:
should it contain any?

```suggestion
  @callback validate(conn :: Plug.Conn.t() | any()) :: {:ok, context :: map()} | {:error, atom()}
```

Reply:
In this case i've already changed it.